nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -424,48 +478,19 @@ define([
          * Returns a structured initial state of the cell.
          */
         function getInitialState() {
-            return {
-                fileType: {
-                    selected: 'fastq',
-                    completed: {
-                        fastq: false,
-                        sra: true
-                    }
-                },
-                tab: {
-                    selected: 'configure',
-                    tabs: {
-                        configure: {
-                            enabled: true,
-                            visible: true,
-                        },
-                        viewConfigure: {
-                            enabled: false,
-                            visible: false
-                        },
-                        info: {
-                            enabled: true,
-                            visible: true
-                        },
-                        jobStatus: {
-                            enabled: true,
-                            visible: true
-                        },
-                        results: {
-                            enabled: false,
-                            visible: true
-                        },
-                        error: {
-                            enabled: false,
-                            visible: false
-                        }
-                    }
-                },
-                action: {
-                    name: 'runApp',
-                    disabled: true
-                }
+            // load current state from state list
+            // modify to handle file types panel
+            let state = States[0].ui;
+            let fileTypeState = {
+                completed: {}
             };
+            for (const fileType of Object.keys(typesToFiles)) {
+                fileTypeState.completed[fileType] = false;
+            }
+            fileTypeState.selected = Object.keys(typesToFiles)[0];
```

Comment:
What happens if there are several file types in `typesToFiles`? Would it be better to ensure that the selected file type is picked in a reproducible manner?